Detectors/MUON/MCH/Simulation/src/Digitizer.cxx

```diff
@@ -186,14 +186,14 @@ void Digitizer::mergeDigits(const std::vector<Digit> digits, const std::vector<o
   int i = 0;
   while (i < indices.size()) {
     int j = i + 1;
-    while (j < indices.size() && (sortedDigits(i).getPadID() == sortedDigits(j).getPadID())) {
+    while (j < indices.size() && ( (100000 * sortedDigits(i).getDetID() + sortedDigits(i).getPadID()) == (100000 * sortedDigits(j).getDetID() + sortedDigits(j).getPadID()))) {
```

Comment:
can you DRY (Dont Repeat Yourself) this part by having the 100k mult. operation done in a function 

(also, do I understand correctly that this operation is just needed for sorting, but is in no way an attempt to get an absolute digit ID ?)

Reply:
Ok, it is elapsed in a function.
It is needed to get order. one could use this number also instead of the two integers, detId, padID, but this is not the case in the current code.